### ISSUE TITLE
Legend to bottom

### ISSUE THREAD

**Issue by krlmlr** [jonocarroll/ggeasy#26]
Is this "easy enough" already? That's my `#2` Google query when I ggplot, `#1` being "rotate x labels".

**Comment by jonocarroll** [comment]
Should be more or less straightforward... How about

```
ggplot(mtcars, aes(hp, mpg)) + 
    geom_point(aes(col = factor(cyl))) + 
    theme(legend.position = "bottom")
```

with a shortcut 

```
ggplot(mtcars, aes(hp, mpg)) + 
    geom_point(aes(col = factor(cyl))) + 
    easy_legend_at("bottom")
```

Alternatively the currently accepted values of `("none", "left", "right", "bottom", "top")`?

**Comment by jonocarroll** [comment]
closes #24
